doc/source/reference/window.rst

```diff
@@ -85,3 +85,14 @@ Base class for defining custom window boundaries.
    :toctree: api/
 
    api.indexers.BaseIndexer
+
+Forward-looking fixed window indexer
```

Comment:
I don't think you need to add a whole new section for this. Just place `api.indexers.FixedForwardWindowIndexer` right below `api.indexers.BaseIndexer`

Reply:
That looks better, thanks. Still learning how our docs are built, sorry.